build/images/base-windows/README.md

```diff
@@ -10,9 +10,15 @@ cd build/images/base-windows
 GO_VERSION=$(head -n 1 ../deps/go-version)
 CNI_BINARIES_VERSION=$(head -n 1 ../deps/cni-binaries-version)
 NANOSERVER_VERSION=$(head -n 1 ../deps/nanoserver-version)
+WIN_OVS_VERSION=$(head -n 1 ../deps/ovs-version-windows)
 WIN_BUILD_TAG=$(echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5sum| head -c 10)
+WIN_BUILD_OVS_TAG=$(echo $NANOSERVER_VERSION-$WIN_OVS_VERSION)
 docker build -t antrea/base-windows:$WIN_BUILD_TAG --build-arg GO_VERSION=$GO_VERSION --build-arg CNI_BINARIES_VERSION=$CNI_BINARIES_VERSION --build-arg NANOSERVER_VERSION=$NANOSERVER_VERSION .
 docker push antrea/base-windows:$WIN_BUILD_TAG
+cd -
+cd build/images/ovs
+docker build --target windows-ovs -f Dockerfile.windows -t antrea/windows-ovs:$WIN_BUILD_OVS_TAG --build-arg WIN_OVS_VERSION=$WIN_OVS_VERSION --build-arg NANOSERVER_VERSION=$NANOSERVER_VERSION .
+docker push antrea/windows-ovs:$WIN_BUILD_OVS_TAG
```

Comment:
why windows-ovs's build command is added to base-windows's README?

Reply:
Do not see any windows manual build command elsewhere, shall it be added to
build/images/ovs/Readme.md ?

Reply:
ovs/README is the right place for the commands. It already has the linux version command.
I don't think people would look for ovs's build command in base-windows's README.